src/components/FocusTrap/WIDE_LAYOUT_INACTIVE_SCREENS.ts

```diff
@@ -33,6 +33,7 @@ const WIDE_LAYOUT_INACTIVE_SCREENS: string[] = [
     SCREENS.WORKSPACE.REPORT_FIELDS,
     SCREENS.WORKSPACE.EXPENSIFY_CARD,
     SCREENS.WORKSPACE.DISTANCE_RATES,
+    SCREENS.WORKSPACE.EXPENSIFY_CARD,
```

Comment:
We added this in line 34. Let's remove this one

Reply:
fixed